music21/romanText/tsvConverter.py

```diff
@@ -336,10 +335,14 @@ def tabToM21(self):
                 # otherwise, we assume self.local_key is already a pitch and
                 # pass it through unchanged
                 localKeyNonRoman = self.local_key
-            thisEntry = roman.RomanNumeral(combined, localKeyNonRoman)
+            thisEntry = roman.RomanNumeral(
+                combined,
+                localKeyNonRoman,
```

Comment:
Don't know how `getLocalKey()` operates but could it be that a `roman.Minor67Default` value could be needed here, as well, since the `.FLAT` behaviour also applies to DCML localkeys (if the globalkey is minor)?

Reply:
Yes, that is now implemented---see lines 953--954. Thanks for your keen eye though :)